com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -374,25 +374,16 @@ void NotifyAgentDone(DoneReason doneReason)
             ReloadPolicy();
         }
 
-        /// <summary>
-        /// Updates the type of behavior for the agent.
-        /// </summary>
-        /// <param name="behaviorType"> The new behaviorType for the Agent.</param>
-        public void SetBehaviorType(BehaviorType behaviorType)
```

Comment:
Removed this in favor of just setting the behavior parameters and letting that update the policy.

Reply:
Should we do the same wit SetModel ?

Reply:
I think SetModel still has a slight advantage if you're changing more than 1 thing - if you change 2 fields, you generate the policy twice.